pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -408,8 +408,10 @@ func TestDesiredRouterDeploymentSpecAndNetwork(t *testing.T) {
 	ic.Spec.Logging = &operatorv1.IngressControllerLogging{
 		Access: &operatorv1.AccessLogging{
 			Destination: operatorv1.LoggingDestination{
-				Type:      operatorv1.ContainerLoggingDestinationType,
-				Container: &operatorv1.ContainerLoggingDestinationParameters{},
+				Type: operatorv1.ContainerLoggingDestinationType,
```

Comment:
You can add this later, but this section should be to test the default.

Reply:
This comment is for L519, will repost there.

Reply:
But it does apply to the addition of MaxLength in this section.  This should only be the default, not a changed value in MaxLength.

Reply:
ok, understood

Reply:
This is setting up the default for testing.  Don't add MaxLength here.